.github/workflows/check-format.yml

```diff
@@ -10,7 +10,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - run: npm install
+      # prettier has no dependencies that need to be installed so we can skip this step
+      # - run: npm install
```

Comment:
Optional, but speeds up the whole action by ~50% (from above 1 minute down to ~30s)